src/main/java/seedu/address/model/person/Analytics.java

```diff
@@ -0,0 +1,227 @@
+package seedu.address.model.person;
+
+import java.util.Date;
+
+/**
+ * Represents the analytics of a LoanRecords object.
+ */
+public class Analytics {
+
+    private int numLoans; // total number of loans
+    private int numOverdueLoans; // total number of overdue loans
+    private int numActiveLoans; // total number of active loans
+
+    private float propOverdueLoans; // proportion of loans that are overdue over active loans
```

Comment:
Comment does not match implementation, which suggests it is proportion of overdue loans over all loans.

Reply:
Thanks, it's fixed.